packages/node_modules/@webex/internal-plugin-metrics/src/metrics.js

```diff
@@ -3,12 +3,42 @@
  */
 
 import {WebexPlugin} from '@webex/webex-core';
-import {deprecated} from '@webex/common';
 
+import BrowserDetection from '../../common/src/browser-detection';
```

Comment:
This should be imported using `@webex/common` not file pathed, this causes issues when bundling